proposals/stage-1/PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -18,100 +18,140 @@ Projects must be proposed via GitHub. Project proposals submitted to the _$FOUND
+
+In order to support their active development, projects in the Growth stage have a higher level of access to marketing and other resources, which will be agreed upon and reviewed on a yearly basis. A project's progress toward its growth plan goals will be reviewed on a yearly basis, and the CPC may ask the project to move to the At Large stage if progress on the plan drops off or stalls. 
 
 **Examples**
+
 1. Projects that are on their way or very likely to become Top Level Projects.
+1. Projects that have developed new growth targets or other community metrics for success.
+1. Projects that are looking to create a lifecycle plan (maintainership succession, contributor programs, version planning, etc.)
+1. Projects that need more active support from the Foundation in the form of marketing or CPC mentorship in order to reach their goals. 
 
 **Expectations**
-Projects can remain in the Incubating stage indefinitely, but they are generally expected to graduate within two years.
+
+Projects in the Growth Stage are generally expected to move out of the Growth stage within two years. Depending on their growth plans, projects may cycle through At Large, Growth, or Impact stage as needed.  
 
 **Acceptance Criteria**
-To be considered for Incubation Stage, the project must meet the Sandbox requirements as well as the following:
- * Document that it is being used successfully in production by at least three independent end users which, in the CPC’s judgement, are of adequate quality and scope.
- * Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+
+To be considered for Growth Stage, the project must meet the At Large requirements as well as the following:
+
+ * Development of a growth plan, to be done in conjunction with their project mentor(s) at the CPC.
+ * Document that it is being used successfully in production by at least two independent end users which, in the CPC’s judgement, are of adequate quality and scope.
  * Demonstrate a substantial ongoing flow of commits and merged contributions.
+ * Demonstrate that the current level of community participation is sufficient to meet the goals outlined in the growth plan.
  * Since these metrics can vary significantly depending on the type, scope and size of a project, the CPC has final judgement over the level of activity that is adequate to meet these criteria.
- * Receive a two-thirds supermajority vote of the CPC to move to Incubation Stage. 
+ * Receive a two-thirds supermajority vote of the CPC to move to Growth Stage. 
```

Comment:
I apologize if we’ve gone over this before—I know @rubys has mentioned it—in the ASF, incubating projects are not yet official members. 

I think that’s maybe a good idea, as it would considerably reduce the overhead involved in getting started (transfer of copyright etc) and avoid being stuck in foundationland if things didn’t work out (once the project’s copyright is owned by the nonprofit, it can only be transferred to another nonprofit, or something like that). 

that sounds incompatible with the idea of projects moving fluidly through this stage from others, of course...

Reply:
so we have that concept captured in the "Commons.js" idea on #44 & other places, if I remember correctly. For this proposal I wanted to focus just on the projects that would officially be in the foundation (and I believe at the CNCF, Sandbox projects are in the foundation). The commons.js program would be the appropriate spot for projects needing some help and wanting to get a sense of what it would be like to work with the foundation. But to your point, we need to draft (or plan to draft) proposals for 1) the commons.js program itself and 2) the requirements/process for projects to leave if they so choose.

Reply:
I'd agree that addressing the aspect raised by @boneskull through commons.js is the right way to go as opposed to another membership level.

Reply:
Housekeeping: I'm opening issues to create stage-0 proposals for the commons.js program & draft exit language to capture these concerns